src/Message/SNS/MessageInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Kfz24\QueueBundle\Message\SNS;
+
+/**
+ * Interface MessageInterface
+ * @package Kfz24\QueueBundle\Message\SNS
+ */
+interface MessageInterface
+{
+    /**
+     * @return int
+     */
+    public function getVersion(): int;
+
+    /**
+     * @return string
+     */
+    public function getType(): string;
```

Comment:
We are missing the actual message content! :email:

Reply:
MessageInterface is the content of `message` property